src/components/PhoneNumberInput/lazy.js

```diff
@@ -0,0 +1,18 @@
+import { h, Component } from 'preact'
+
+let Empty = () => <div>Not here</div>
+
+class PhoneNumberInputLazy extends Component {
+  constructor(props){
+    super(props)
+    this.state = {component: Empty }
+
+    import(/* webpackChunkName: "cross-device" */ './index.js').then(component => {
```

Comment:
so this triggers a download of another bundle?

How does the browser know what to download?

Reply:
this is not lazy loading, only code splitting. It's up to to the developer to still include the secondary js bundle in the html. The advantage is that they can load after the page load (by using `async`). Lazy loading requires to host the js bundles.